configs/projects/_shared-client-tools-runtime.rb

```diff
@@ -107,11 +107,9 @@
 # Common deps
 proj.component "runtime-client-tools"
 
-if proj.settings[:openssl_version]
-  proj.component "openssl-#{proj.openssl_version}"
-elsif platform.name =~ /^redhatfips-.*/
+if(proj.settings[:openssl_version] == '1.1.1k')
```

Comment:
Before any macOS 13 changes for client tools runtimes, we always included the `openssl-1.1.1-fips` component on all redhatfips platforms (both 7 & 8):

https://github.com/puppetlabs/puppet-runtime/blob/aee9e9da2e63811f8940c4f2b1854ec50ce2b33f/configs/projects/_shared-client-tools-runtime.rb#L105-L106

This PR fixes rhel8 fips, but not rhel7 fips, because `openssl_version` is only set to `1.1.1k` on the former, but not the latter: https://github.com/puppetlabs/puppet-runtime/blob/4d56b3d434d89a74243bc91d30c8f3968602692c/configs/projects/_shared-client-tools-runtime.rb#L28-L29

Here's the most recently client tools 2021.7.x on rhel7 showing the [wrong openssl component](https://jenkins-platform.delivery.puppetlabs.net/view/puppet-runtime/view/client-tools-runtime-2021.7.x/job/platform_client-tools-runtime_runtime-vanagon-packaging_2021.7.x/135/BUILD_TARGET=redhatfips-7-x86_64,SLAVE_LABEL=k8s-worker)

```
18:14:13 touch openssl-1.1.1-patch
18:14:13 [ -d openssl-1.1.1v ] || mkdir -p openssl-1.1.1v
18:14:13 export PATH="/opt/pl-build-tools/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/usr/local/bin" && \
18:14:13 cd openssl-1.1.1v && \
18:14:13  ./Configure --prefix=/opt/puppetlabs/client-tools --libdir=lib --openssldir=/opt/puppetlabs/client-tools/ssl shared no-asm linux-x86_64 no-camellia no-ec2m no-md2 no-ssl3 -I/opt/puppetlabs/client-tools/include -I/opt/pl-build-tools/include -L/opt/puppetlabs/client-tools/lib -L/opt/pl-build-tools/lib -Wl,-rpath=/opt/puppetlabs/client-tools/lib -Wl,-z,relro
18:14:13 Configuring OpenSSL version 1.1.1v (0x1010116fL) for linux-x86_64
```